docs/toc_en.yml

```diff
@@ -61,6 +61,10 @@ nav:
     - 'IPv4': 'data_types/domains/ipv4.md'
     - 'IPv6': 'data_types/domains/ipv6.md'
 
+- 'Database Engines':
```

Comment:
Maybe let's try to make it like this?

Engines
-> Database
---> ClickHouse
---> MySQL
-> Table
---> ...

Reply:
?

![image](https://user-images.githubusercontent.com/2968367/59918039-39c20800-942c-11e9-9a61-bd87fb4853d7.png)

Reply:
Well, I'm still trying to keep the number of top level sections low.
You can keep it as is and we'll figure out how to improve it later.

Reply:
I think we need to find a new structure for whole documentation. I planned to do this in autumn. But I can force this task after my vacations.